src/app/pages/notifications-page/notifications.page.html

```diff
@@ -1,18 +1,23 @@
 <app-header [currentPageTitle]="headerPageTitle"></app-header>
-
+<ion-header>
+  <ion-toolbar>
+    <ion-title>Notifications</ion-title>
+  </ion-toolbar>
+</ion-header>
 <ion-content>
   <ng-container *ngIf="notifications.length > 0; else noNotifications">
     <ion-list>
-      <ion-item *ngFor="let notification of notifications" appLongPress
-        (longPress)="onDeleteNotification(notification.id)" (click)="onShowMoreInfo(notification)"
-        [class.long-press]="isPressed">
+      <ion-item *ngFor="let notification of notifications"
+                appLongPress (longPress)="onDeleteNotification(notification.id)"
+                (click)="onShowMoreInfo(notification)"
+                [class.long-press]="isPressed">
```

Comment:
Where is `isPressed` defined?

Reply:
isPressed is like isClicked or onSubmit. Its a built in event listener. I am not sure if it is exclusive to angular.